images/iqsharp-base/Dockerfile

```diff
@@ -20,7 +20,7 @@ RUN apt-get update && \
                        # Dependencies for the Quantum Development Kit.
                        # Note that we install them here to minimize the number
                        # of layers.
-                       libgomp1 \
+                       libomp-7-dev \
```

Comment:
Why is this needed in the iqsharp-base Dockerfile? That indicates to me that even with static linking, there's a new requirement passed onto users.

Reply:
Static linking attempt has failed. I failed to find the `libomp.a` static library, it wasn't installed as a part of libomp package (I didn't try to build `libomp.a` from sources). 
Also, IIRC, there was some hypothetical risk with licensing, if we provide libomp together with our native simulator. But _installing_ the libomp had no any licensing issues.
> Why is this needed in the iqsharp-base Dockerfile?

Because the native simulator dynamic library is built on _ubuntu-latest_, where libomp is already installed, but the  iqsharp-base Docker image (that uses the native simulator dynamic library) is run on Debian Buster (Debian 10), where the libomp is not installed. So I added the installation of libomp on Debian Buster.

Reply:
@kuzminrobin can we try `libomp5-7`? That should just be the libomp runtime.